src/debug_menu.cpp

```diff
@@ -62,1059 +62,1062 @@
 namespace debug_menu
 {
 
-    class mission_debug
-    {
-        private:
+class mission_debug
+{
+    private:
         // Doesn't actually "destroy" the mission, just removes assignments
         static void remove_mission( mission &m );
-        public:
+    public:
         static void edit_mission( mission &m );
         static void edit( player &who );
         static void edit_player();
         static void edit_npc( npc &who );
         static std::string describe( const mission &m );
-    };
+};
 
-    int player_uilist()
-    {
-        std::vector<uilist_entry> uilist_initializer;
-
-        uilist_initializer.emplace_back( uilist_entry( 8, true, 'M', _( "Mutate" ) ) );
-        uilist_initializer.emplace_back( uilist_entry( 10, true, 's', _( "Change all skills" ) ) );
-        uilist_initializer.emplace_back( uilist_entry( 11, true, 'l', _( "Learn all melee styles" ) ) );
-        uilist_initializer.emplace_back( uilist_entry( 12, true, 'r', _( "Unlock all recipes" ) ) );
-        uilist_initializer.emplace_back( uilist_entry( 13, true, 'p', _( "Edit player/NPC" ) ) );
-        uilist_initializer.emplace_back( uilist_entry( 23, true, 'd', _( "Damage Self" ) ) );
-        uilist_initializer.emplace_back( uilist_entry( 28, true, 'a', _( "Set automove route" ) ) );
-
-        int action = uilist( _( "Info..." ), uilist_initializer );
-        if( action < 0 ) {
-            return debug_menu::debug_menu_uilist();
-        } else {
-            return action;
-        }
+int player_uilist()
+{
+    std::vector<uilist_entry> uilist_initializer;
+
+    uilist_initializer.emplace_back( uilist_entry( 8, true, 'M', _( "Mutate" ) ) );
```

Comment:
This can be replaced with list-initialization, become `const` and also `static` (meaning the lifetime here):
```c++
const std::vector<uilist_entry> uilist_initializer = {
     { .... }
};
```